src/commands/import.js

```diff
@@ -145,7 +145,8 @@ class ImportCommand extends FaunaCommand {
           collection,
           path,
           Boolean(this.flags['dry-run']),
-          this.warn
+          this.warn,
+          Boolean(this.flags['allow-retries'])
```

Comment:
IMO we don't need a flag for this - we're retrying things that we know didn't persist and are retryable. We should just do this for the user without needing configuration.